### PR TITLE
Reset user and session on setup

### PR DESCRIPTION
**Pull request by SenNeonoveNoci**

### CODE REVIEW

File: www/SmartlookPlugin.js [hunk 1]
```diff
@@ -138,15 +140,29 @@ exports.setupAndStartRecording = function (options, successCallback, errorCallba
         arguments.push(UNDEFINED_RENDERING_MODE)
     }
 
+    if (checkBooleanOption("setupAndStartRecording", "startNewSession", options, renderingModeAllowedValues, errorCallback, false)) {
```
Comment (pavel-kroh-smartlook) [inline www/SmartlookPlugin.js:144]: renderingModeAllowedValues ?

File: www/SmartlookPlugin.js [hunk 2]
```diff
@@ -138,15 +140,29 @@ exports.setupAndStartRecording = function (options, successCallback, errorCallba
...
+        arguments.push(options["startNewSession"])
+    } else {
+        arguments.push(false)
+    }
+
+    if (checkBooleanOption("setupAndStartRecording", "startNewSessionAndUser", options, renderingModeAllowedValues, errorCallback, false)) {
```
Comment (pavel-kroh-smartlook) [inline www/SmartlookPlugin.js:150]: renderingModeAllowedValues ?

File: www/SmartlookPlugin.js (hunk 1 above)
Comment (SenNeonoveNoci) [reply www/SmartlookPlugin.js]: Opraveno.

File: www/SmartlookPlugin.js (hunk 2 above)
Comment (SenNeonoveNoci) [reply www/SmartlookPlugin.js]: Opraveno.

**Pull request merged**